Automattic/jetpack
# CRM: Add backend action Invoice Set Status and Invoice Event Manager

This PR adds the action Invoice Set Status, and the needed event manager for invoices.

## Proposed changes:
* Add Invoice Set Status Action
* Add Invoice Event Manager

### Other information:


## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3331
https://github.com/Automattic/zero-bs-crm/issues/3195

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:






* Check the code
* Check the coverage of our new tests
* Run the tests and make sure they are passing
